recipe/meta.yaml

```diff
@@ -25,25 +24,32 @@ requirements:
   run:
     - python
     - numpy >=1.17.3
-    - scipy >=1.3.2
+    - scipy >=1.5.0
     - scikit-learn >=1.0.2
     - joblib >=1.1.1
-  run_constrained:
     - threadpoolctl >=2.0.0
+  run_constrained:
     # intel-openmp 2023.1 and llvm-openmp 14.0.6 appear not be compatible
     # leading to segfault in test_pairwise_distances_reduction.py::test_sqeuclidean_row_norms[42-float64-8-5-100]
     - intel-openmp <2023.1.0  # [osx and x86_64]
+    - pandas >=1.0.5
+    - tensorflow >=2.4.3
+    - keras >=2.4.3
 
 test:
   requires:
     - pip
     - pytest >=5.0.1
     - pytest-timeout
+    - black >=23.3.0
+    - flake8 >=3.8.2
+    - mypy >=1.3.0
```

Comment:
```suggestion
```
Not needed

Reply:
I added them as optional dependencies since `intel-openmp` was already mentioned. But I do get that intel-openmp is a special case.

Reply:
This is the test section... black, mypy and flake8 aren't used in our tests.

Reply:
my mistake. I added them because they were listed as test dependencies but indeed they have nothing to do with purest.